core/src/bls_sigverify/bls_sigverifier.rs

```diff
@@ -1478,6 +1521,7 @@ mod tests {
             message_sender,
             consensus_metrics_sender,
             Arc::new(AlpenglowLastVoted::default()),
+            Arc::new(MigrationStatus::post_migration_status()),
```

Comment:
Should we maybe add one unittest to make sure we are dropping packets correctly pre and post migration?